style/index.html

```diff
@@ -504,8 +459,74 @@
           </div>
           <div id='transfer-funds'>
             <div class='header'>
+              <div class='back button'>
+                <i class='fa fa-chevron-left'></i>
+              </div>
               <div class='title'>Transfer Funds</div>
             </div>
+            <div class='row'>
+              <div class='col-xs-6'>
+                <div class='from-column'>
+                  <div class='subhead'>
+                    <i class='fa fa-sign-out'></i>
+                    Transferring from Savings
+                  </div>
+                  <div class='account'>
+                    <div class='eqn current-balance'>
+                      <div class='text'>Balance</div>
+                      <div class='unit'>SC</div>
+                      <div class='amt'>32.001&nbsp;</div>
+                    </div>
+                    <div class='eqn transfer-amt'>
+                      <div class='text'>Transfer</div>
+                      <div class='unit'>SC</div>
+                      <div class='amt input field-transfer'>
+                        <i class='icon fa fa-pencil'></i>
+                        2.142
+                      </div>
+                    </div>
+                    <div class='eqn final-balance'>
+                      <div class='text'>Final Balance</div>
+                      <div class='unit'>SC</div>
+                      <div class='amt input field-final-balance'>
+                        <i class='icon fa fa-pencil'></i>
+                        28.859
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              </div>
+              <div class='col-xs-6 right-column'>
+                <div class='to-column'>
+                  <div class='subhead'>
+                    <i class='fa fa-sign-in'></i>
+                    Transferring to Address
+                  </div>
+                  <div class='address'>
+                    <div class='eqn'>
+                      <div class='text address-title'>SC Address</div>
+                      <div class='amt input field-address'>
+                        <i class='icon fa fa-pencil'></i>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div class='control'>
+              <div class='summary'>
+                Transferring
+                <div class='amt'>2.142 SC</div>
+                from
+                <div class='from'>Savings Account</div>
+                to
+                <div class='to'>0cc175bd71c7...</div>
+              </div>
+              <div class='button send-money'>
+                <i class='fa fa-send'></i>
+                Send Funds
+              </div>
+            </div>
```

Comment:
All the html and css is generated, here's the relevant line in haml...

```haml
.control
        .summary
            Transferring
            .amt 2.142 SC
            from
            .from Savings Account
            to
            .to 0cc175bd71c7...
```

In haml, everything is by default a div. In this code, the display type of the divs are changed so that they are displayed inline or side by side (this may have been a point of confusion). This makes the haml really readable and hopefully understandable. When changing the project to pure html we'd want to convert these divs to spans or another inline element because there would be no haml to "look clean".